pippo-demo/src/main/java/ro/fortsoft/pippo/demo/SimpleApplication.java

```diff
@@ -122,7 +138,7 @@ public void handle(Request request, Response response, RouteHandlerChain chain)
 
                 Map<String, Object> model = new HashMap<>();
                 model.put("greeting", message);
-                response.render("hello", model);
+                response.html().render("hello", model);
```

Comment:
As you say, it's not necessary because we default to it with `render()`.  I was just being specific.  When mixing API handlers and page handlers in the same class being explicit about return types may make errors more obvious.